src/rust/grapl-web-ui/src/lib.rs

```diff
@@ -27,6 +28,8 @@ pub fn run(config: config::Config) -> Result<Server, std::io::Error> {
             jsonwebtoken_google::Parser::new(&config.google_client_id),
         ));
         let graphql_endpoint = Data::new(config.graphql_endpoint.clone());
+        let plugin_registry_client = Data::new(Mutex::new(config.plugin_registry_client.clone()));
```

Comment:
fwiw i believe our tonic Clients are cheap to clone already, no lock needed?

Reply:
PluginRegistryServiceClient::create_plugin takes a mutable self, which I can't get behind the web::Data without a Mutex. https://docs.rs/actix-web/latest/actix_web/web/struct.Data.html.

Reply:
ah got it

Reply:
Yes-ish, I'd need `plugin_registry_client.get_ref().clone()`, but regardless I think you're right that I should just do that instead. Thanks!